tests/test_pii_detection.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Please update the scripts name as `test_guardrails_pii_detection.sh`, so the test can be auto triggered.

Reply:
Updated